tests/parametric/test_dynamic_configuration.py

```diff
@@ -402,3 +402,41 @@ def test_capability_tracing_custom_tags(self, library_env, test_agent, test_libr
         """Ensure the RC request contains the custom tags capability.
         """
         test_agent.wait_for_rc_capabilities([Capabilities.APM_TRACING_CUSTOM_TAGS])
+
+    @missing_feature(context.library in ["cpp", "dotnet", "golang"])
+    @parametrize("library_env", [{**DEFAULT_ENVVARS}])
+    def test_capability_tracing_enabled(self, library_env, test_agent, test_library):
+        """Ensure the RC request contains the tracing enabled capability."""
+        test_agent.wait_for_rc_capabilities([Capabilities.APM_TRACING_ENABLED])
+
+    @missing_feature(context.library in ["cpp", "dotnet", "golang"])
+    @parametrize(
+        "library_env", [{**DEFAULT_ENVVARS}, {**DEFAULT_ENVVARS, "DD_TRACE_ENABLED": "false"},],
+    )
+    def test_tracing_client_tracing_enabled(self, library_env, test_agent, test_library):
```

Comment:
I think we should break the test out into its own class so that libraries can pin the precise version that this was introduced rather than grouping it with the other settings